fandogh_cli/utils.py

```diff
@@ -63,10 +63,14 @@ def get_window_width():
         return None
 
 
-def parse_key_values(key_values):
+def parse_key_values(key_values, read_env):
     env_variables = {}
     for env in key_values:
         (k, v) = env.split('=', 1)
+        if read_env and os.environ.get(k, default=None) is not None:
```

Comment:
The priority of passed value for parameter should be higher than the set env variable